Week 2 - Arrays/scrabble/scrabble.c

```diff
@@ -0,0 +1,72 @@
+#include <cs50.h>
+#include <ctype.h>
+#include <stdio.h>
+
+int getIndex(char key);
+int getValue(char key);
+int countPoints(string word);
+
+int SIZE = 26;
+char KEYS[26] = {'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M',
```

Comment:
I use it to map amount of times we meet each letter in a key and then to "translate" the text. Maybe there is a better solution, but I didn't find it

Reply:
sure, this way it works, it just less resistant to "human factor". Instead of this array you can use ASCII table itself that was introduced in the lecture (see the picture below). You can rely on a fact that each letter corresponds to specific number in the table and use letter offset instead of indexing your own custom array. 
If my idea is not clear here I will share it in details on Monday meeting 🙏🏾 
![image](https://github.com/khaletska/CS50/assets/13601748/0f264c1e-5351-4b54-a2f1-b283b3dbeb78)